docs/custom-queries.md

```diff
@@ -0,0 +1,25 @@
+# Making Custom Calls to Graph
+
+The Graph SDK attempts to enable all available scenarios through Microsoft Graph. There are times, however, through errors or custom Graph functionality, that makes calling the desired endpoint is not possible through the provided requests and builders.
```

Comment:
Remove the 'is' in 'that makes calling the desired endpoint **is** not possible.'